net/vpn-policy-routing/Makefile

```diff
@@ -4,8 +4,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpn-policy-routing
-PKG_VERSION:=0.3.2
-PKG_RELEASE:=20
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=0
```

Comment:
I always get told that releases should start with `1`.

Reply:
Thanks, I'll push a fix within a few days.

Reply:
Ping?

Reply:
Sorry,  an issue was discovered and I didn't have the time to look into during the week. If I can't push the update by tomorrow I'll close these.

Reply:
No rush, I just thought you forgot about this.